### ISSUE TITLE
slowonly_imagenet_pretrained_r50_4x16x1_120e_gym99_rgb.py  lr_config setting

### ISSUE THREAD

**Issue by a-github-learner** [open-mmlab/mmaction2#2215]
In slowonly_imagenet_pretrained_r50_4x16x1_120e_gym99_rgb.py, lr_config was set with total_epochs=120 as:` lr_config = dict(policy='step', step=[90, 110])`
I would like to ask how many steps should I set as I set `total_epochs=300` or `total_epochs=240`

**Comment by a-github-learner** [comment]
I train the module with config below and the module is overfitting while training. I am wondering if the params mentioned before cause this problem ? Otherwise, it just a parameters that doesn't matter? What's the real reason make the module overfitting? BTW, I use only two GPUS to train. Thank you and looking forward to your reply.
`
_base_ = [
    '../../_base_/models/slowonly_r50.py', '../../_base_/default_runtime.py'
]

# dataset settings
dataset_type = 'RawframeDataset'
data_root = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/subaction_frames'
data_root_val = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/subaction_frames'
ann_file_train = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_train_frame.txt'
ann_file_val = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_val_frame.txt'
ann_file_test = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_val_frame.txt'
img_norm_cfg = dict(
    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_bgr=False)
train_pipeline = [
    dict(type='SampleFrames', clip_len=4, frame_interval=16, num_clips=1),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(type='RandomResizedCrop'),
    dict(type='Resize', scale=(224, 224), keep_ratio=False),
    dict(type='Flip', flip_ratio=0.5),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs', 'label'])
]
val_pipeline = [
    dict(
        type='SampleFrames',
        clip_len=4,
        frame_interval=16,
        num_clips=1,
        test_mode=True),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(type='CenterCrop', crop_size=224),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs'])
]
test_pipeline = [
    dict(
        type='SampleFrames',
        clip_len=4,
        frame_interval=16,
        num_clips=10,
        test_mode=True),
    dict(type='RawFrameDecode'),
    dict(type='Resize', scale=(-1, 256)),
    dict(type='ThreeCrop', crop_size=256),
    dict(type='Normalize', **img_norm_cfg),
    dict(type='FormatShape', input_format='NCTHW'),
    dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['imgs'])
]
data = dict(
    videos_per_gpu=24,#24
    workers_per_gpu=2,#2
    test_dataloader=dict(videos_per_gpu=1),
    train=dict(
        type=dataset_type,
        ann_file=ann_file_train,
        data_prefix=data_root,
        pipeline=train_pipeline),
    val=dict(
        type=dataset_type,
        ann_file=ann_file_val,
        data_prefix=data_root_val,
        pipeline=val_pipeline),
    test=dict(
        type=dataset_type,
        ann_file=ann_file_test,
        data_prefix=data_root_val,
        pipeline=test_pipeline))
evaluation = dict(
    interval=1, 
    save_best='mean_class_accuracy',
    metrics=['top_k_accuracy', 'mean_class_accuracy'])
workflow = [('train', 1),('val',1)] # [('train', 1),('val',1)]
# optimizer
optimizer = dict(
    type='SGD', lr=0.0075, momentum=0.9,
    weight_decay=0.0001)  # 0.03 lr is used for 8 gpus
optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))
# learning policy
lr_config = dict(policy='step', step=[90, 110])
total_epochs = 300#120
dist_params = dict(backend='nccl')
log_config = dict(
    interval=20, hooks=[
        dict(type='TextLoggerHook'),
    ])
log_level='INFO'
# runtime settings
work_dir = './work_dirs/recognition/slowonly_imagenet_pretrained_r50_4x16x1_120e_gym99_rgb'
resume_from=None
find_unused_parameters = False

`

**Comment by a-github-learner** [comment]
I train the module with config below and the module is overfitting while training. I am wondering if the params mentioned before cause this problem ? Otherwise, it just a parameters that doesn't matter? What's the real reason make the module overfitting? BTW, I use only two GPUS to train. 

Thank you and looking forward to your reply.


```
base = [
'../../base/models/slowonly_r50.py', '../../base/default_runtime.py'
]
 
dataset settings

dataset_type = 'RawframeDataset'
data_root = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/subaction_frames'
data_root_val = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/subaction_frames'
ann_file_train = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_train_frame.txt'
ann_file_val = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_val_frame.txt'
ann_file_test = '/media/6t/XJY_FILES/Gym/mmaction2-master/tools/data/gym/annotations/gym99_val_frame.txt'
img_norm_cfg = dict(
mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_bgr=False)
train_pipeline = [
dict(type='SampleFrames', clip_len=4, frame_interval=16, num_clips=1),
dict(type='RawFrameDecode'),
dict(type='Resize', scale=(-1, 256)),
dict(type='RandomResizedCrop'),
dict(type='Resize', scale=(224, 224), keep_ratio=False),
dict(type='Flip', flip_ratio=0.5),
dict(type='Normalize', **img_norm_cfg),
dict(type='FormatShape', input_format='NCTHW'),
dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
dict(type='ToTensor', keys=['imgs', 'label'])
]
val_pipeline = [
dict(
type='SampleFrames',
clip_len=4,
frame_interval=16,
num_clips=1,
test_mode=True),
dict(type='RawFrameDecode'),
dict(type='Resize', scale=(-1, 256)),
dict(type='CenterCrop', crop_size=224),
dict(type='Normalize', **img_norm_cfg),
dict(type='FormatShape', input_format='NCTHW'),
dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
dict(type='ToTensor', keys=['imgs'])
]
test_pipeline = [
dict(
type='SampleFrames',
clip_len=4,
frame_interval=16,
num_clips=10,
test_mode=True),
dict(type='RawFrameDecode'),
dict(type='Resize', scale=(-1, 256)),
dict(type='ThreeCrop', crop_size=256),
dict(type='Normalize', **img_norm_cfg),
dict(type='FormatShape', input_format='NCTHW'),
dict(type='Collect', keys=['imgs', 'label'], meta_keys=[]),
dict(type='ToTensor', keys=['imgs'])
]
data = dict(
videos_per_gpu=24,https://github.com/open-mmlab/mmaction2/pull/24
workers_per_gpu=2,https://github.com/open-mmlab/mmaction2/pull/2
test_dataloader=dict(videos_per_gpu=1),
train=dict(
type=dataset_type,
ann_file=ann_file_train,
data_prefix=data_root,
pipeline=train_pipeline),
val=dict(
type=dataset_type,
ann_file=ann_file_val,
data_prefix=data_root_val,
pipeline=val_pipeline),
test=dict(
type=dataset_type,
ann_file=ann_file_test,
data_prefix=data_root_val,
pipeline=test_pipeline))
evaluation = dict(
interval=1,
save_best='mean_class_accuracy',
metrics=['top_k_accuracy', 'mean_class_accuracy'])
workflow = [('train', 1),('val',1)] # [('train', 1),('val',1)]

optimizer
optimizer = dict(
type='SGD', lr=0.0075, momentum=0.9,
weight_decay=0.0001) # 0.03 lr is used for 8 gpus
optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))

learning policy
lr_config = dict(policy='step', step=[90, 110])
total_epochs = 300#120
dist_params = dict(backend='nccl')
log_config = dict(
interval=20, hooks=[
dict(type='TextLoggerHook'),
])
log_level='INFO'

runtime settings
work_dir = './work_dirs/recognition/slowonly_imagenet_pretrained_r50_4x16x1_120e_gym99_rgb'
resume_from=None
find_unused_parameters = False```

**Comment by eliethesaiyan** [comment]
@a-github-learner , have you tried to do reduce the learning rate? 0.0075/2(gpus)?
